src/sgame/components/SpikerComponent.cpp

```diff
@@ -242,14 +245,14 @@ bool SpikerComponent::Fire() {
 			if (!fire) {
 				continue;
 			}
+			++launched;
 
 			G_SpawnMissile(
 				MIS_SPIKER, self, self->s.origin, dir, nullptr, G_FreeEntity,
 				level.time + (int)(1000.0f * SPIKE_RANGE / (float)BG_Missile(MIS_SPIKER)->speed));
 		}
 	}
-
-	restUntil = level.time + COOLDOWN;
+	restUntil = level.time + COOLDOWN * ( sg_spiker_experiments.Get() ? launched * sg_spiker_missiles.Get() : 1 );
```

Comment:
It seems this PR is not up to date with whatever changes you have tested locally. Could you update to your latest, functional version?

Reply:
done. The problem was basically about the PR being months old, and in the meantime, we merged the data submodule. Which means I'd be able to put the other changes I initially did inside this PR, actually.

Reply:
actually only 3 weeks old, but the initial patch is from february